brightener.cpp

```diff
@@ -5,22 +5,22 @@ ImageBrightener::ImageBrightener(Image& inputImage) : m_inputImage(inputImage) {
 
 int ImageBrightener::BrightenWholeImage() {
 	// For brightening, we add a certain grayscale (25) to every pixel.
-	// While brightening, some pixels may cross the max brightness. They are
-	// called 'attenuated' pixels
+	// While brightening, some m_pixels may cross the max brightness. They are
+	// called 'attenuated' m_pixels
 	int attenuatedPixelCount = 0;
-	for (int x = 0; x < m_inputImage.rows; x++) {
-		for (int y = 0; y < m_inputImage.columns; y++) {
-			if (m_inputImage.pixels[x * m_inputImage.columns + y] > (255 - 25)) {
+	for (int x = 0; x < m_inputImage.m_rowSize; x++) {
+		for (int y = 0; y < m_inputImage.m_columnSize; y++) {
+			if (m_inputImage.m_pixels[x * m_inputImage.m_columnSize + y] > (255 - 25)) {
 				++attenuatedPixelCount;
-				// m_inputImage.pixels[x * m_inputImage.rows + 1 + y] = 255;
-				m_inputImage.pixels[x * m_inputImage.columns + y] = 255;
+				// m_inputImage.m_pixels[x * m_inputImage.m_rowSize + 1 + y] = 255;
```

Comment:
Don't leave commented code in commits. If they are needed later, they will be in the git history